roles/openshift_prometheus_operator/files/assets/prometheus/rules/etcd3.rules.yaml

```diff
@@ -0,0 +1,123 @@
+groups:
+- name: ./etcd3.rules
```

Comment:
how are you discovering etcd nodes, and how do you authN/authZ with them?

Reply:
> ...how do you authN/authZ with them?

Good point. In our production clusters the etcd targets would return http 403.

Reply:
Haven't done anything with etcd, and we may not for this.

Reply:
That is, I indiscriminately pulled in all the upstream kube-prometheus rules and service monitors knowing some would work and some wouldn't, with the intent of spot checking the controller manager (most important) and fixing any others we care about in followups.